client/index.html

```diff
@@ -9,6 +9,7 @@
 	<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 	<meta name="mobile-web-app-capable" content="yes">
 	<meta name="referrer" content="no-referrer">
+	<meta name="theme-color" content="#455164">
```

Comment:
The only problem as you are describing it, is that the color will look slightly out of place on non default themes. We can't really make this customizable per theme until we get packages or theme settings.